retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp

```diff
@@ -445,15 +445,15 @@ QVariant RsGxsForumModel::textColorRole(const ForumModelPostEntry& fmpe,int /*co
     if( (fmpe.mPostFlags & ForumModelPostEntry::FLAG_POST_IS_MISSING))
         return QVariant(mTextColorMissing);
 
+    if(fmpe.mPostFlags & ForumModelPostEntry::FLAG_POST_IS_PINNED)
```

Comment:
Why pinned post couldn't be in (un)read shape (via color change) at all?

Reply:
It doesn't mean much that a pinned post is unread or not, and the place to put the unread icon is occupied by the pin icon. Anyway, it's still possible to use bold of course.

Reply:
From my point of view whether post (any) read or not is quite important. When there are many pinned posts and one of them not yet read, guessing what exactly is, is not something good for me (whether i just have not time to read yet, or i saved it in unread state for the purpose).

Reply:
Agreed. Thx for checking.

Reply:
Thx for agreeding.